bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -443,7 +486,22 @@
                                 }
                             }
                             return cachedProtocols;
+                        case "$methods":
+                            if (cachedNativeMethodNames === null) {
+                                cachedNativeMethodNames = [];
+
+                                // Fill cachedMethodNames
+                                this.keys();
+
+                                cachedNativeMethodNames = Object.keys(cachedMethods).filter(m => m.match(/^(\+|-)/));
+                            }
+                            return cachedNativeMethodNames;
                         default:
+                            if (handle.isNull()) {
+                                return function() {
+                                    return 0;
```

Comment:
Wouldn't it be better to return `ObjC.nil` instead? So you can keep calling methods on every object returned and it will "just work".